types/urlencode/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "semicolon": false
```

Comment:
Please don't turn off linter rules, it makes typings much easier to read if they're uniform across the project.